Source/Parallelization/WarpXComm.cpp

```diff
@@ -348,36 +349,34 @@ WarpX::SyncCurrent ()
     }
 }
 
-/** \brief Fills the values of the current on the coarse patch by
- *  averaging the values of the current of the fine patch (on the same level).
- */
 void
 WarpX::SyncCurrent (const std::array<const amrex::MultiFab*,3>& fine,
-                    const std::array<      amrex::MultiFab*,3>& crse,
-                    int refinement_ratio)
+                    const std::array<      amrex::MultiFab*,3>& coarse,
+                    int const refinement_ratio)
 {
```

Comment:
Could you add a `BL_PROFILE` here?

Reply:
I think there might be instrumentation already for this, this member is called from `WarpX::SyncCurrent ()`

Reply:
And the file name.